utils/ansible.py

```diff
@@ -0,0 +1,281 @@
+from distutils.sysconfig import get_python_lib
+from os import listdir, mkdir, path, remove
+from shutil import copy, copyfile, rmtree
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from fauxfactory import gen_alphanumeric
+from utils import conf
+from utils.providers import get_crud
+
+from git import Repo
+from yaml import load, dump
+
+
+yml_path = path.join(path.dirname(__file__) + "/manageiq_ansible_module/")
+library_path = get_python_lib()
+basic_yml_path = path.dirname(__file__) + "/ansible_conf/"
```

Comment:
**REQUIRED** we should avoid tests writing into anything but the tmpdir managed by pytest, i can assist in handling that detail via a fixture